tools/check-templates

```diff
@@ -43,11 +44,19 @@ def check_html_templates(templates, modified_only):
     # The casperjs files use HTML5 (whereas Zulip prefers XHTML), and
     # there are also cases where Casper deliberately uses invalid HTML,
     # so we exclude them from our linter.
+    templates_Casperjs_Filtered = filter(
+        lambda fn: ('casperjs' not in fn),
+        templates) # type: (Iterable[str])
+
     templates = filter(
-        lambda fn: ('base.html' not in fn) and ('casperjs' not in fn),
-        templates)
+        lambda fn: ('base.html' not in fn),
+        templates_Casperjs_Filtered)
+
     templates = sorted(list(templates))
 
+    templates_Casperjs_Filtered = sorted(list(templates_Casperjs_Filtered))
```

Comment:
Our coding standard is to use snake_case, so we'd case the var name here something like `templates_casper_js_filtered`.  I would also consider a name like 'templates_to_check_for_ids' in this particular example.